Note-Each rule can have up to 10 case-sensitive prefixes and upto 10 … (#90306)
Azure/azure-content: articles/storage/blobs/lifecycle-management-policy-configure.md

---

A lifecycle management policy must be read or written in full. Partial updates are not supported.

> > [!NOTE]

Reviewer: ```suggestion

> [!NOTE]
```

> > Each rule can have up to 10 case-sensitive prefixes and upto 10 blob index tag conditions.

Reviewer: Each rule can have up to ten case-sensitive prefixes, as well as up to ten blob index tag conditions.

Author: Thanks @tamram . Looks good !

Author: Hi @tamram -Each rule can have up to ten case-sensitive prefixes, as well as up to ten blob index tag conditions- This looks good. I accept these, kindly proceed with merge if it's okay.

A lifecycle management policy must be read or written in full. Partial updates are not supported.

> [!NOTE]

> > Each rule can have up to 10 case-sensitive prefixes and upto 10 blob index tag conditions.

Reviewer: ```suggestion
> Each rule can have up to ten case-sensitive prefixes and up to ten blob index tag conditions.
```

> > Each rule can have up to ten case-sensitive prefixes and up to ten blob index tag conditions.

Reviewer 2: ```suggestion
> Each rule can have up to 10 case-sensitive prefixes and up to 10 blob index tag conditions.
```
Minor style edits
